anta/input_models/security.py

```diff
@@ -43,6 +53,95 @@ class IPSecConn(BaseModel):
     """The IPv4 address of the destination in the security connection."""
 
 
+class APISSLCertificate(BaseModel):
+    """Model for an API SSL certificate."""
+
+    model_config = ConfigDict(extra="forbid")
+    certificate_name: str
+    """The name of the certificate to be verified."""
+    expiry_threshold: int
+    """The expiry threshold of the certificate in days."""
+    common_name: str
+    """The common subject name of the certificate."""
```

Comment:
The capitalized words have a meaning in certificate world (CN)
```suggestion
    """The Common Name of the certificate."""
```

Reply:
Updated